mathics/builtin/list/eol.py

```diff
@@ -475,17 +483,18 @@ def apply(self, expr, evaluation):
 
 class FirstCase(Builtin):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/FirstCase.html</url>
+
     <dl>
       <dt> FirstCase[{$e1$, $e2$, ...}, $pattern$]
-      <dd>gives the first $ei$ to match $pattern$, or $Missing[\"NotFound\"]$ if none matching pattern is found.
+      <dd>gives the first $ei$ to match $pattern$, or $Missing["NotFound"]$ if none matching pattern is found.
 
       <dt> FirstCase[{$e1$,$e2$, ...}, $pattern$ -> $rhs$]
       <dd> gives the value of $rhs$ corresponding to the first $ei$ to match pattern.
       <dt> FirstCase[$expr$, $pattern$, $default$]
       <dd> gives $default$ if no element matching $pattern$ is found.
 
-      <dt>FirstCase[$expr$, $pattern$, $default$, $levelspec$] \
-      <dd>finds only objects that appear on levels specified by $levelspec$.
+      <dt>FirstCase[$expr$, $pattern$, $default$, $levelspec$]       <dd>finds only objects that appear on levels specified by $levelspec$.
```

Comment:
Line join added here.

Reply:
Notice that this is due to a wrong backslash in the master version.